lib/client/ipam.go

```diff
@@ -577,28 +577,26 @@ func (c ipams) RemoveIPAMHost(host *string) error {
 	return nil
 }
 
-func (c ipams) hostBlockPairs(pool common.IPNet) (map[string]string, error) {
```

Comment:
I made it a pointer since `common.ParseCIDR` returns pointers, but it's kind of a silly reason.